샘플 프로젝트 실행 가능한 구조로 변경

sungjin-woo opened the pull request:
## 수정 내용

- 이전 코드 수행을 위해서 우선 Xcode 프로젝트로 생성

## 스크린샷

![Simulator Screenshot - iPhone 15 Pro Max - 2024-05-17 at 19 05 17](https://github.com/worksmobile/XbICalendar/assets/5136232/e5df780f-fa8c-440d-9ef7-8b4f40a1a3e6)

sungjin-woo committed:
샘플 프로젝트 실행 가능한 구조로 변경

- 이전 코드 수행을 위해서 우선 Xcode 프로젝트로 생성

sungjin-woo committed:
Git ignore 설정 추가

IMCHO commented:
이전 pr 머지되었습니다!
해당 pr까지 정리되면 새 버전 릴리즈하도록 하겠습니다.

sungjin-woo commented:
@IMCHO 님,
브랜치 리베이스 했습니다.
승인 해주시면 머지 진행하겠습니다.

IMCHO commented on Package.swift:
```diff
@@ -29,14 +29,14 @@ let package = Package(
-            ],
-            path: "XbICalendar/XbICalendarExample"
-        ),
+//        .executableTarget(
```
spm 상에서 빌드할수는 없는걸까요?

sungjin-woo replied:
빌드는 되는데 실행이 안되더라구요. ㅠ

<img width="1766" alt="스크린샷 2024-05-21 오후 3 31 02" src="https://github.com/worksmobile/XbICalendar/assets/5136232/6a9dedb6-5ebd-4a85-9c97-b92f466b5beb">

원인을 알수가 없어서 우선은 Xcode Project를 살린 상태에요.

sungjin-woo replied:
기존에 XcodeProject에 앱에 대한 설정이 있는데요.
이 부분을 Package.swift 로 옮겨야 할 것 같은데요.

`products.executable`이나 `executableTarget.linkerSettings`을 통해서 먼가 설정을 주입할수 있으면 될 것 같은데 
아직은 제가 공부가 부족한것 같아요.

IMCHO replied:
main 파일에서 `UIApplicationMain` 생성 시 크래쉬가 나는거 같은데
많이 검색을 해봤는데 제가 생각했던것처럼 spm 내에서 샘플앱 빌드를 할 수 있지 않은 거 같네요...

지금 스토리보드 파일이 중복되는거 같은데 ViewControllers 내 디렉토리에 있는 스토리보드만 제거 부탁드리겠습니다!

sungjin-woo committed:
리뷰사항 반영

- 중복된 파일 삭제
  - Main.storyboard

## 기타 수정

- 샘플프로젝트 deployment target 변경
- Deprecated 된 UIAlertView로 인한 크래시 이슈 수정

sungjin-woo commented:
# 추가수정

# 리뷰사항 반영

- 중복된 파일 삭제
  - `Main.storyboard`

## 기타 수정

- 샘플프로젝트 deployment target 변경: 15.0
- Deprecated 된 UIAlertView로 인한 크래시 이슈 수정
  - 오류 처리 추가

IMCHO reviewed (approved):
수고하셨습니다. 감사합니다!
